ksql-engine/src/main/java/io/confluent/ksql/KsqlExecutionContext.java

```diff
@@ -98,6 +98,8 @@ ExecuteResult execute(
       KsqlConfig ksqlConfig,
       Map<String, Object> overriddenProperties);
 
+  List<PersistentQueryMetadata> getPersistentQueries();
```

Comment:
Couple of points on this one:

- nit: how about some java docs, given every other method in this interface has them?
- nit: would you mind moving this 'persistentQuery' related method next to the other persistent query methods, eg. next to `getPersistentQuery`.
- suggestion: to keep the interface succinct, if we're adding `getPersistentQueries()` then can we remove `numberOfPersistentQueries` and just have client code call `getPersistentQueries().size()`.